Tests/TypologyTests/ASTTests.swift

```diff
@@ -43,4 +43,24 @@ final class ASTTests: XCTestCase {
       variables: [TypeVariable(value: tVar)]
     ))
   }
+
+  func testPosition() throws {
+    let functions = try """
+        func first(_ x: String) -> String {
```

Comment:
please add comments before a function declaration here. After looking at SwiftSyntax docs I have a feeling that its `position` includes comments, which is treated as trivia, as far as I remember